src/viur/core/bones/relational.py

```diff
@@ -1027,11 +1027,8 @@ def refresh(self, skel: "SkeletonInstance", name: str) -> None:
         if not skel[name] or self.updateLevel == RelationalUpdateLevel.OnValueAssignment:
             return
 
-        for idx, lang, value in self.iter_bone_value(skel, name):
-            if value is None:
-                continue
-
-            if value["dest"]:
+        for _, _, value in self.iter_bone_value(skel, name):
+            if value and value["dest"]:
```

Comment:
```suggestion
            if not (value and value["dest"]):
                continue
```
And indent everything underneath to the left, I think that always looks nicer, don't you?

Reply:
@ArneGudermann we can do it on a follow-up PR. I already wrote code but I cannot test it correctly as #1403 is not there right now, and I don't have time now.